sdk/src/wallet/account/operations/syncing/addresses/output_ids/alias_foundry.rs

```diff
@@ -104,18 +104,16 @@ impl Account {
     ) -> crate::wallet::Result<Vec<OutputId>> {
         log::debug!("[SYNC] get_foundry_output_ids");
         // Get alias outputs, so we can then get the foundry outputs with the alias addresses
-        let alias_output_responses = self.get_outputs(alias_output_ids.iter().cloned().collect()).await?;
+        let alias_outputs_with_meta = self.get_outputs(alias_output_ids.iter().cloned().collect()).await?;
```

Comment:
Can the alias_ids not be `copied` instead?

Reply:
Yes, Thanks, fixed